cirq/contrib/quimb/mps_simulator.py

```diff
@@ -29,35 +29,60 @@
 from cirq.sim import simulator
 
 
+class MPSOptions:
+    def __init__(
+        self,
+        method: str = 'svds',
+        max_bond: Optional[int] = None,
+        cutoff_mode: str = 'rsum2',
+        cutoff: float = 1e-6,
+        sum_prob_atol: float = 1e-3,
+    ):
+        """Numerical options for the simulation.
+
+        Args:
+            method: How to split the tensor. Refer to the Quimb documentation
```

Comment:
Recommend adding a link here to the quimb docs.

Reply:
Done.